README.md

```diff
@@ -15,13 +11,11 @@ Additional description about the project and its features.
 
 ## Live Demo
```

Comment:
- [ ] It seems that this section of your README is empty, kindly remove the line or add the live demo, you can use GitHub Pages or Githack for that, as it's not a mandatory requirement, choosing to delete the line will not block your approval.